force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -183,14 +205,12 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<Integer> createAndPopulateList() {
         // Create a new list of integers
+        List<Integer> intList = new List<Integer>();
+        intList.add(1);
+        intList.add(2);
+        intList.add(3);
```

Comment:
You can also load up the list statically by just declaring it with these 3 values.